packages/backend-common/src/reading/GithubUrlReader.ts

```diff
@@ -22,7 +22,7 @@ import {
   ScmIntegrations,
 } from '@backstage/integration';
 import { RestEndpointMethodTypes } from '@octokit/rest';
-import fetch, { RequestInit, Response } from 'node-fetch';
+import { fetch, Response, RequestInit } from 'undici';
```

Comment:
@punkle nope :joy: I would start from a fork of this repo and get things setup so that you can reproduce the error locally with the current setup, and then replace it in the `GithubUrlReader` and see what happens on your machine to test it.

Reply:
I don't think that we would want to just ship it for the GIthubUrlReader you see either.

Reply:
haha nice ok just wanted to make sure we're on the same level 😅 , and I think it's because we would need to use ]
`import { stream } from 'undici'` as the request method not `fetch`. The `whatwg-fetch` spec does not have any specification on `response.body` being a stream so theres a separate API for it as it tries to keep with the web spec.